### DISCUSSION TITLE
How to close *italics* this situation

### DISCUSSION THREAD

**Discussion by FreedomClannad** [remarkjs/remark#1283] [Q&A] [answered]
I want to turn off *italics* in react-markdown in this case

**Comment by wooorm** [comment]
?

**Reply by FreedomClannad** [reply]
There is a format in markdown * xxx * which will become italicized. The effect I want is to turn off this situation. Just for * xxx * this situation

**Reply by wooorm** [reply]
There’s also a format for code, GitHub reads markdown, so you have to use code to explain your code.

`react-markdown` is for markdown too. You should use it for markdown. Not for other things.

**Reply by FreedomClannad** [reply]
Because I am going to show the smiles in chemistry, the molecular structure of * cccc2c( * )cccc2c1. Since there are two close * in it, the middle c1ccc2c( becomes italicized, so I just want to close the situation where there is only * xxx * .Because the data passed cannot be transferred in *.

**Reply by FreedomClannad** [reply]
This is the relevant DEMO address  https://stackblitz.com/edit/stackblitz-starters-cp29kf?file=src%2FApp.tsx

**Reply by ChristianMurphy** [reply, marked as answer]
@FreedomClannad as noted below https://github.com/orgs/remarkjs/discussions/1283#discussioncomment-8306297 creating a new markdown flavor is often problematic.
Instead considering leveraging markdown escapes
```markdown
\*c1ccc2c(\*)cccc2c1
```
runnable example: https://stackblitz.com/edit/stackblitz-starters-zndy8x

Or for more in-depth chemistry formatting consider using MathJax with MChem https://mhchem.github.io/MathJax-mhchem/ and https://github.com/remarkjs/remark-math

**Comment by ChristianMurphy** [comment]
Welcome @FreedomClannad! 👋 
react-markdown and remark build off micromark which can be customized https://github.com/micromark/micromark?tab=readme-ov-file#extending-markdown

That said, both human authors and most AI chatbots know how to write markdown (CommonMark).
Italics are part of CommonMark, changing the language will cause confusion with no benefits.
Just because it is possible to turn off italics and other constructs does not make it a good idea.
